Fatras/include/ActsFatras/Physics/ElectroMagnetic/detail/GaussianMixture.hpp

```diff
@@ -84,7 +85,7 @@ struct GaussianMixture {
       sigma2 *= (1. - (1. - epsilon) * sigma1square) / epsilon;
     }
     // return back to the
-    return M_SQRT2 * std::sqrt(sigma2) * gaussDist(generator);
```

Comment:
We do probably have `std::sqrt(2)` in the code as well, did you have a look ?

Reply:
yes, I checked all. I also ran `modernize-use-std-numbers` with clang-tidy to detect more cases.